pkg/config/parameters.go

```diff
@@ -484,6 +484,13 @@ type ListenerParameters struct {
 
 	// SocketOptions is used to set socket options for listeners.
 	SocketOptions SocketOptions `yaml:"socket-options"`
+
+	// Defines the limit on number of HTTP requests that Envoy will process from a single
+	// connection in a single I/O cycle. Requests over this limit are processed in subsequent
+	// I/O cycles. Can be used as a mitigation for CVE-2023-44487 when abusive traffic is
+	// detected. Configures the http.max_requests_per_io_cycle Envoy runtime setting. The default
+	// value when this is not set is no limit.
+	MaxRequestsPerIOCycle *uint32 `yaml:"max-requests-per-io-cycle"`
```

Comment:
Missing this description also in [site/content/docs/main/configuration.md](site/content/docs/main/configuration.md)?

Reply:
yep good catch 👍🏽